bliss.ipynb

```diff
@@ -0,0 +1,221 @@
+{
+ "metadata": {
```

Comment:
Is the `jmd` file supposed to replace this?

Reply:
Yes, sorry, I'll delete that right away.